app/main/views.py

```diff
@@ -3,6 +3,13 @@
 from . import main
 from ..lib.authentication import requires_authentication
 from ..models import Service
+from flask import abort
+from flask import make_response
+from flask import render_template
+from flask import request
```

Comment:
these can be collapsed into `from flask import abort, make_response, render_template, request`